### PR TITLE
Disable the Openshift log message for discovery

### PR DESCRIPTION
**Pull request by mangelajo**
When this is ran on non-openshift that log message appears on console,
when it's actually irrelevant. All discovery mechanism are tried
one after another until we find one that works.

```
[majopela@bluehat submariner-operator]$ bin/subctl show networks
I0626 11:06:21.463397    3263 openshift4.go:47] Attempted network discovery for OpenShift4, no clusternetworks CRD
    Discovered network details:
        Network plugin:  weave-net
        Service CIDRs:   [100.93.0.0/16]
        Cluster CIDRs:   [10.243.0.0/16]
```

### CODE REVIEW

File: pkg/discovery/network/openshift4.go [hunk 1]
```diff
@@ -43,8 +42,8 @@ func discoverOpenShift4Network(dynClient dynamic.Interface) (*ClusterNetwork, er
 	crClient := dynClient.Resource(openshift4clusterNetworkGVR)
 
 	cr, err := crClient.Get("default", metav1.GetOptions{})
+
 	if err != nil {
-		klog.Info("Attempted network discovery for OpenShift4, no clusternetworks CRD")
```
Comment (tpantelis) [inline pkg/discovery/network/openshift4.go:47]: Perhaps we should at least log this to debug (including `err`) rather than silence it completely.

Comment (maayanf24) [reply pkg/discovery/network/openshift4.go:47]: In any case, it should be symmetric to all of the discovery mechanism

Comment (mangelajo) [reply pkg/discovery/network/openshift4.go:47]: Good comments.

Comment (tpantelis) [reply pkg/discovery/network/openshift4.go:47]: Actually we just need to check for `IsNotFound` error. This is addressed as part of https://github.com/submariner-io/submariner-operator/pull/581 which should obviate the need for any changes in this PR.

**Comment by mangelajo** [comment]
This would be better handled by: https://github.com/submariner-io/submariner-operator/pull/581

**Pull request closed without merging**